src/towncrier/newsfragments/626.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed section names not having enough underline characters for emojis.
```

Comment:
Maybe having this read like a release notes 

the current version looks like a commit message.

This information is for end users... and the end result is that they can now use emoji in category names :)

```suggestion
For reStructuredText format, you can now use emojis in category names.
In previous versions, the generated underline was not matching the title width. 
```

Reply:
Done